packages/ui/src/common/components/page/Sidebar/Breadcrumbs/BreadcrumbsList.tsx

```diff
@@ -1,17 +1,27 @@
 import React from 'react'
+import { useHistory } from 'react-router'
 import styled from 'styled-components'
+import useBreadcrumbs from 'use-react-router-breadcrumbs'
 
-import { BreadcrumbsItem, BreadcrumbsItemProps } from './BreadcrumbsItem'
+import { knownRoutes, excludePaths } from '../../../../model/breadcrumbs'
+
+import { BreadcrumbsItem } from './BreadcrumbsItem'
 
 export interface BreadcrumbsListProps {
-  crumbs: Array<BreadcrumbsItemProps>
+  objectName?: string
 }
 
-export function BreadcrumbsList({ crumbs }: BreadcrumbsListProps) {
+export function BreadcrumbsList({ objectName }: BreadcrumbsListProps) {
+  const { location } = useHistory()
+  const routesList = objectName ? [...knownRoutes, { path: location.pathname, breadcrumb: objectName }] : knownRoutes
```

Comment:
Now I see ;) So the `objectName` is `breadcrumb`. So the better name would be either `activeBreadcrumb`, `lastBreadcrumb` or just `breadcrumb` (ps. I still don't know what this does so I might shooting blind with the names).